egs/aspire/s5/local/multi_condition/aspire_data_prep.sh

```diff
@@ -3,31 +3,34 @@
 # Apache 2.0.
 set -e
 stage=0
-aspire_data=/export/corpora5/ASpIRE/
+# Set this to somewhere where you want to put your aspire data, or where
+# someone else has already put it.  You'll want to change this
+# if you're not on the CLSP grid.
+aspire_data=/export/corpora/LDC/LDC2017S21/IARPA-ASpIRE-Dev-Sets-v2.0/data ;; # JHU
```

Comment:
I think this variable should be set at the run.sh level and passed in as a command line argument.